.github/workflows/workflow.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - add-latest-in-workflow
```

Comment:
Do you intend to merge this change, or is this just here to try the workflow out?

Reply:
This will be removed before landing :)